scripts/requirements.txt

```diff
@@ -0,0 +1,3 @@
+PyYAML
+requests
+git+https://github.com/OpenBAS-Platform/release.git@improvement/new_release_workflow
```

Comment:
Should be the master branch ?

Reply:
Good catch ! Fixing it here and on the other repo